File: Python/Beginners/validacao_nota.py
# Caio Cesar 01/12/23
# BEE 1117

contador = 0
soma = 0
while contador < 2:
    nota = float(input())
    if 0 <= nota <= 10:
        contador += 1
        soma += nota
    else:
        print("nota invalida")

media = soma / 2

print(f"media = {media:.2f}")

File: C++/Beginner/validacao_nota.cpp
// Caio Cesar 29/11/23
// BEE 1048

#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    int contador = 0;
    float soma, nota, media;

    while(contador < 2){
        cin >> nota;
        if(nota >= 0 && nota <= 10){
            soma += nota;
            contador++;
        }else{
            cout << "nota invalida" << endl;
        }
    }

    media = soma / 2.0;

    cout << "media = " << media << setprecision(2) << fixed << endl;

    return 0;
}